docs/excel/custom-functions-debugging.md

```diff
@@ -50,7 +50,6 @@ Debugging for custom functions can be accomplished by multiple means, depending
 ### For Windows Or Mac
 
 1. Open your custom functions root project folder in VS Code.
-2. Select **Terminal | Run Task** and type or select Watch.
```

Comment:
@mscharlock You should put this line back. For custom functions, you need to run both Watch and Dev Server tasks.  

For a TaskPane WebView only, you don't need to run the Watch task.

Reply:
Gotcha, ok!